pandas/io/tests/test_packers.py

```diff
@@ -333,7 +333,8 @@ def setUp(self):
             'E': [0., 1, Timestamp('20100101'), 'foo', 2.],
             'F': [Timestamp('20130102', tz='US/Eastern')] * 2 +
                  [Timestamp('20130603', tz='CET')] * 3,
-            'G': [Timestamp('20130102', tz='US/Eastern')] * 5
+            'G': [Timestamp('20130102', tz='US/Eastern')] * 5,
+            'H': Categorical(['a', 'b', 'c', 'd', 'e'])
```

Comment:
I added `'I': Categorical(['a', 'b', 'c', 'd', 'e'], ordered=True)` and it passes without any implementation, so I guess `assert_series_equal` is not sufficient? Should I create a new separate test which does the encode/decode and checks that the `ordered` value is correctly preserved?